contracts/extensions/StreamingFeeSplitExtension.sol

```diff
@@ -0,0 +1,207 @@
+/*
+    Copyright 2022 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+    
+    SPDX-License-Identifier: Apache License, Version 2.0
+*/
+
+pragma solidity 0.6.10;
+pragma experimental "ABIEncoderV2";
+
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
+import { SafeMath } from "@openzeppelin/contracts/math/SafeMath.sol";
+
+import { PreciseUnitMath } from "@setprotocol/set-protocol-v2/contracts/lib/PreciseUnitMath.sol";
+import { ISetToken } from "@setprotocol/set-protocol-v2/contracts/interfaces/ISetToken.sol";
+
+import { BaseGlobalExtension } from "../lib/BaseGlobalExtension.sol";
+import { IDelegatedManager } from "../interfaces/IDelegatedManager.sol";
+
+import { IStreamingFeeModule } from "../interfaces/IStreamingFeeModuleV2.sol";
```

Comment:
This new interface is necessary because I need a `IStreamingFeeModule` which
- uses the `ISetToken` interface that the DelegatedManager/BaseGlobalExtension is using ("@setprotocol/set-protocol-v2/contracts/interfaces/ISetToken.sol";)
- has `accrueFee` in the interface

The set-protocol-v2 `IStreamingFeeModule` uses the correct `ISetToken` but does not have `accrueFee`
The set-v2-strategies `IStreamingFeeModule` uses a local `ISetToken` but does have `accrueFee`

So I created set-v2-strategies `IStreamingFeeModuleV2` which uses the correct `ISetToken` and has `accrueFee`

This brings up a general issue with the coexisting repos set-v2-strategies and set-protocol-v2
- Ideally, interfaces (and all common contracts) should be defined once overall in set-protocol-v2 and set-v2-strategies. This forces common usage in the coexisting repos. 
- Interfaces should include all functions and structs, not just the ones needed up to the current commit. This ensures new development will be supported by the interface. 

I propose we fix this by enforcing the above two conditions and refactoring previous code to follow this.

Reply:
Short term fix for this is to update set-protocol-v2 `IStreamingFeeModule` 

set-v2-strategies `IStreamingFeeModule` cannot update `ISetToken` because that breaks previous tests related to the `FeeSplitExtension`